src/sql/workbench/api/browser/mainThreadNotebook.ts

```diff
@@ -258,8 +260,13 @@ class SessionManagerWrapper implements azdata.nb.SessionManager {
 class SessionWrapper implements azdata.nb.ISession {
 	private _kernel: KernelWrapper;
 	constructor(private _proxy: Proxies, private sessionDetails: INotebookSessionDetails) {
```

Comment:
while you're here, if you could put an _ before sessionDetails that'd be great 😄